src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/MonitorThreadContainer.java

```diff
@@ -46,13 +46,18 @@ public static MonitorThreadContainer getInstance() {
     public static synchronized MonitorThreadContainer getInstance(IExecutorServiceInitializer executorServiceInitializer) {
         if (singleton == null) {
             singleton = new MonitorThreadContainer(executorServiceInitializer);
+            CLASS_USAGE.getAndSet(0);
```

Comment:
When you don't store the value after, is get needed? Can you just set?

Reply:
Makes sense to just use set here.